packages/preview-middleware-client/src/adp/init-dialogs.ts

```diff
@@ -3,11 +3,17 @@ import type Dialog from 'sap/m/Dialog';
 
 /** sap.ui.core */
 import Fragment from 'sap/ui/core/Fragment';
-import type UI5Element from 'sap/ui/core/Element';
+import UI5Element from 'sap/ui/core/Element';
 
 /** sap.ui.rta */
 import type RuntimeAuthoring from 'sap/ui/rta/RuntimeAuthoring';
 
+/** sap.ui.fl */
+import FlUtils from 'sap/ui/fl/Utils';
+
+/**  */
```

Comment:
Either remove this or add the lib name (optional)

Reply:
Done